### PR TITLE
[as7326-56x] Modify to check eeprom by pre_pddf_init.sh

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

#### Why I did it
Modify to check eeprom by pre_pddf_init.sh
#### How I did it
Add pre_pddf_init.sh to code base.
#### How to verify it
Test eeprom and other pddf cmd that are fine
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-accton/debian/sonic-platform-accton-as7326-56x.postinst [hunk 1]
```diff
@@ -2,27 +2,7 @@
...
-    # syseeprom is at the i2c address 0x57. Change the PDDF JSON file
-    sed -i 's@"topo_info": {"parent_bus": "0x0", "dev_addr": "0x56", "dev_type": "24c04"},@\
-        "topo_info": {"parent_bus": "0x0", "dev_addr": "0x57", "dev_type": "24c02"},@g' \
-        /usr/share/sonic/device/x86_64-accton_as7326_56x-r0/pddf/pddf-device.json
-    sync
-fi
-
 depmod -a
-systemctl enable as7326-platform-handle_mac.service
-systemctl start as7326-platform-handle_mac.service
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/debian/sonic-platform-accton-as7326-56x.postinst:25]: Why this is removed? Please include it.

Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/debian/sonic-platform-accton-as7326-56x.postinst:25]: Add handle_mac  back to the .posinst file.

File: platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pre_pddf_init.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    use_57_eeprom=false
+    echo "The board has system EEPROM at I2C address 0x56."
+fi
+
+if $use_57_eeprom ; then
+    echo "The board has system EEPROM at I2C address 0x57."
+    # syseeprom is at the i2c address 0x57. Change the PDDF JSON file
+    if [ -f /usr/share/sonic/device/x86_64-accton_as7326_56x-r0/pddf_support ] && \
+       [ -f /usr/share/sonic/device/x86_64-accton_as7326_56x-r0/pddf/pddf-device.json ]; then
+        sed 's@"topo_info": {"parent_bus": "0x0", "dev_addr": "0x56", "dev_type": "24c04"},@\
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/pre_pddf_init.sh:18]: It should have been "sed -i "
-i is missing here

**Comment by FuzailBrcm** [comment]
Can you take care of that "sed" command comment?

**Comment by jostar-yang** [comment]
> Can you take care of that "sed" command comment?

OK. Done it.

**Review by FuzailBrcm — APPROVED** [review]
Looks good to me

**Pull request merged**